libraries/app/api.cpp

```diff
@@ -162,7 +162,8 @@ namespace graphene { namespace app {
     void network_broadcast_api::broadcast_transaction(const signed_transaction& trx)
     {
        trx.validate();
-       _app.chain_database()->push_transaction(trx);
+       const auto& chain_db = _app.chain_database();
```

Comment:
Remove the '&', it's useless here (chain_database() returns a shared_ptr, not a database reference.
OR dereference the shared_ptr right here, might even be a bit more efficient.